go.mod

```diff
@@ -11,9 +11,9 @@ replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 require (
 	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
 	golang.org/x/sync v0.6.0
-	istio.io/istio v0.0.0-20240127054631-015b39fc798e
-	k8s.io/apimachinery v0.29.0
-	k8s.io/client-go v0.29.0
+	istio.io/istio v0.0.0-20240131085136-068f6475f6b2
```

Comment:
This is a commit from the master branch which cannot be used at the moment. Currently, commits must come from the `release-1.21` branch until 1.21.0 is released and we can move the istio.io tests back to the master branch.

Reply:
I reverted to what is on latest master in this repo. I will come back and add the other example when things are settled/released :)